src/gui/editorwidgets/qgsrelationreferencewidgetwrapper.h

```diff
@@ -19,6 +19,7 @@
 #include "qgseditorwidgetwrapper.h"
 #include "qgis_sip.h"
 #include "qgis_gui.h"
+#include "qgsvectorlayerref.h"
```

Comment:
That's probably a leftover?

Reply:
yes, thanks, I said it needed polishing (ans tests!)

Reply:
sure, just bumped into it.
I hope the tests can be easily adapted.

Reply:
I'm not sure, QgsVectorLayerRef is not exposed to Python so I need a C++ test and there aren't any for the formatters.

Reply:
just copying / renaming the ones you did for widgetwrappers won't work?

Reply:
I'm creating a new `testqgsrelationreferencefieldformatter.cpp` and moving the old test into it.